_rules/audio-transcript-2eb176.md

```diff
@@ -22,27 +27,29 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element that is:
+This rule applies to every [non-streaming](#non-streaming-media-element) `audio` element for which at least one of the following is true:
 
-- playing; or,
-- has a [play button][] that is [visible][] and [included in the accessibility tree][].
+- the element has an [autoplay][] [attribute value][] of `true`; or
```

Comment:
Since autoplay here is referring to the HTML attribute I don't think it needs to link to the definition.